notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>mmm interesting, in reviewnb this paragraph is not rendered exactly the same as in the notebook. </p><p>For the list, would it make more sense to add : instead of /? </p><ol><li>Wide: cross...</li><li>Deep: lower...</li></ol>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Make sense. Also editing the contents into a table per Le's suggestion.